vmhost/hostCore/flags.go

```diff
@@ -0,0 +1,18 @@
+package hostCore
+
+import "github.com/multiversx/mx-chain-core-go/core"
+
+const (
```

Comment:
add a comment here valid for all consts?

Reply:
pushed